src/App.jsx

```diff
@@ -0,0 +1,194 @@
+import { useEffect, useState } from 'react';
+import S from './App.module.css';
```

Comment:
```suggestion
import Style from './App.module.css';
```

Style이라고 정확히 명시해주면 더 좋을 것 같아요!